src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -215,7 +215,7 @@ post_processing()
       }
 for(int i = 0; i<=anatomical_image_filenames.size()-1; i++)
 {  if (!this->anatomical_image_filenames.empty()){
-      this->anatomical_prior_sptr.push_back(read_from_file<TargetT>(anatomical_image_filenames[i]));
+      this->anatomical_prior_sptr.push_back(shared_ptr<TargetT>(read_from_file<TargetT>(anatomical_image_filenames[i])));
```

Comment:
probably won't work due to fun pre-C++`` stuff. I think you'll need a variable, but let's see

Reply:
travis is happy, so I am as well.